www/wip_new_website/content/functional.md

```diff
@@ -53,41 +53,41 @@ In Roc, this will give a compile-time error. Once a name has been assigned to a
 
 A benefit of this design is that it makes Roc code easier to rearrange without causing regressions. Consider this code:
 
-```elm
+```roc
 func = \arg ->
     greeting = "Hello"
     welcome = \name -> "\(greeting), \(name)!"
-    …
+    ...
```

Comment:
It may look better if we make these a comment instead? I think the parser is parsing these as decimal `.` characters. 

I only just realised it is probably a better solution and we may be able to keep the shorter unicode character.